lib/resources/Account.js

```diff
@@ -2,12 +2,40 @@
 
 var StripeResource = require('../StripeResource');
 var stripeMethod = StripeResource.method;
+var utils = require('../utils');
 
 module.exports = StripeResource.extend({
-  path: 'account',
+  // Since path can either be `account` or `accounts`, support both through stripeMethod path
```

Comment:
Sure, I guess I mean that if you're using "stripe.account" as a singleton
to grab your own account info, your context/use for it is probably
different than if you were using "stripe.accounts"--I imagine not everyone
will need to use or care to think about "stripe.accounts", though I've
definitely not thought about this as much as you have.

- Michelle

On Fri, Feb 13, 2015 at 11:48 AM, Brian Krausz <notifications@github.com>
wrote:

> In lib/resources/Account.js
> <https://github.com/stripe/stripe-node/pull/144#discussion_r24691057>:
>
> >
> >  module.exports = StripeResource.extend({
> > -  path: 'account',
> > +  // Since path can either be `account` or `accounts`, support both through stripeMethod path
>
> That being said, permissions will dictate what fields you can see and use.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/stripe/stripe-node/pull/144/files#r24691057>.
>